config/config.go

```diff
@@ -284,25 +284,25 @@ type Node struct {
 		// Interval between price updater calls
 		Interval Duration `validate:"required"`
 		// URLBitfinexV2 is the URL of bitfinex V2 API
-		URLBitfinexV2 string `validate:"required"`
+		URLBitfinexV2 string `validate:"required,url"`
 		// URLCoinGeckoV3 is the URL of coingecko V3 API
-		URLCoinGeckoV3 string `validate:"required"`
+		URLCoinGeckoV3 string `validate:"required,url"`
 		// DefaultUpdateMethod to get token prices
-		DefaultUpdateMethod priceupdater.UpdateMethodType `validate:"required"`
+		DefaultUpdateMethod priceupdater.UpdateMethodType `validate:"required,is-valid-updatemethodtype,is-updatemethodtype-not-static"`
 		// TokensConfig to specify how each token get it's price updated
 		TokensConfig []priceupdater.TokenConfig
 	} `validate:"required"`
 	StateDB struct {
 		// Path where the synchronizer StateDB is stored
-		Path string `validate:"required"`
+		Path string `validate:"required,file"`
 		// Keep is the number of checkpoints to keep
-		Keep int `validate:"required"`
+		Keep int `validate:"required,lt=128"`
```

Comment:
I don't think we should impose an upper bound here.

Reply:
Actually to follow the previous requirement in the code, this should be `gte=128`